dbms/src/Storages/DeltaMerge/ScanContext.h

```diff
@@ -78,7 +78,6 @@ class ScanContext
     std::atomic<uint64_t> mvcc_input_rows{0};
     std::atomic<uint64_t> mvcc_input_bytes{0};
     std::atomic<uint64_t> mvcc_output_rows{0};
-    std::atomic<uint64_t> late_materialization_skip_rows{0};
```

Comment:
After this is removed, can we directly use `dmfile_lm_filter_scanned_rows` and `dmfile_lm_filter_skipped_rows` to check the efficiency of late materialization?

Reply:
Yes.